services/galley/test/integration/API/Teams.hs

```diff
@@ -1216,14 +1233,147 @@ testTeamAddRemoveMemberAboveThresholdNoEvents = do
           ((/= Galley.Types.Teams.Intra.Deleted) . Galley.Types.Teams.Intra.tdStatus)
           (getTeamInternal tid)
 
+testBillingInLargeTeam :: TestM ()
+testBillingInLargeTeam = do
+  (firstOwner, team) <- Util.createBindingTeam
+  refreshIndex
+  o <- view tsGConf
+  g <- view tsGalley
```

Comment:
```suggestion
  opts <- view tsGConf
  galley <- view tsGalley
```

Reply:
Had to rename at few more places, but it is done :)